third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -64,6 +77,13 @@ using RemoteObjectIdTypeRaw = std::u16string;
 // The more convenient type that we use
 using RemoteObjectIdType = WTF::String;
 
+constexpr auto P = recordreplay::Print; // because we are lazy?
```

Comment:
A comment explaining why this is needed would be good.  If you don't know for sure, they a comment along the lines of "needed to do this to get it to compile, not sure why" would be good.